CONTRIBUTING.md

```diff
@@ -110,18 +107,26 @@ After adding new snapshot tests or making UI changes, run
  ./scripts/record_screenshot_tests.sh
 ```
 
-This will generate the latest screenshot. Verify the changes & generated screenshots are as expected and commit the changes.
+This will generate the latest snapshots. Verify the changes & generated snapshots are as expected and commit the changes.
 
 To test changes use
 
 ```
 ./scripts/verify_screenshot_tests.sh
 ```
 
-If the check fails you either need to fix the issue if a change was unintended or record script above instead to update the screenshots.
+If the check fails you either need to fix the issue if a change was unintended or record script above instead to update the snapshots.
+
+### Using CI for generating snapshot
+
+Alternatively, you can use Github Actions CI to generate the snapshots.
+"Record snapshots" command in PR comments will trigger the CI and lead to new snapshot commits pushed to your PR.
+Notice that this run must be approved for external contributors.
+Since CI run cannot trigger CI checks again,
```

Comment:
Maybe clarify here that this means to add an empty commit (and maybe include how to do that in the command line)